integration/deploy/manifest_test.go

```diff
@@ -675,7 +674,7 @@ func createOktetoManifest(dir, content string) error {
 }
 
 func expectImageFoundSkippingBuild(output string) error {
-	if ok := strings.Contains(output, "Skipping build for image for service"); !ok {
```

Comment:
Should we include a test with `--build=false` to make sure that the flag works when set it to false?